lib/board.rb

```diff
@@ -0,0 +1,54 @@
+class Board
+  attr_accessor :cells
+
+  def reset! #Create a new blank Array
+    @cells = Array.new(9, " ")
+  end
+
+  #initialize would create a new board aka cells representing the board
+  def initialize
+    reset!
+  end
+
+  #Defining the display board
+  def display
+    def dash
```

Comment:
Reason: Carried over from "Intro to Ruby"
Resolution: Issue addressed
Comment: Agreed & Closed